3.10/installation-linux.md

```diff
@@ -1,38 +1,53 @@
 ---
 layout: default
-description: To install ArangoDB on Linux visit the Download page of the ArangoDB and download the correct package for your Linux distribution. Follow the instructions to use your favorite package.
-title: Installing ArangoDB on Linux 
+description: Download an installation or tar package, or use a package manager
+title: Installing ArangoDB on Linux
 ---
 Installing ArangoDB on Linux
 ============================
 
-To install ArangoDB on Linux:
+You can install ArangoDB on most common Linux distributions. The basic
+installation steps are:
 
-1. Visit the official [Download](https://www.arangodb.com/download){:target="_blank"} page of the
-   ArangoDB web site and download the correct package for your Linux distribution.
-   You can find binary packages for the most common distributions there. Linux Mint: 
-   please use the corresponding Ubuntu or Debian packages. 
-2. Follow the installation instructions on the _Download_ page to use your
-   favorite package manager for the major distributions. After setting up the ArangoDB
-   repository you can easily install ArangoDB using _yum_, _aptitude_, _urpmi_ or _zypper_.
+1. Visit the official [Download](https://www.arangodb.com/download){:target="_blank"}
+   page of the ArangoDB web site.
 
-{% hint 'info' %}
-In addition to installation packages (distribution dependent) a `tar.gz` archive
-is available starting from version 3.4.0.
-{% endhint %}
+2. Click the logo of the distribution that matches your operating system.
+   If you use Linux Mint, click Ubuntu or Debian.
+
+3. You can choose between different installation methods and packages:
+   - distribution-dependent installation packages (`.rpm`, `.deb`)
+   - tar packages (`tar.gz` archives)
+   - installation via a package manager
+
+4. Installation and tar packages: You may verify the integrity of a download
+   by comparing the SHA256 hash listed on the website with the hash of the file.
+   For example, you can you run `openssl sha256 <filename>` or
+   `sha256sum <filename>` in a terminal.
+
+   Package manager: Package managers verify downloaded packages automatically.
+
+5. Installation packages: Run the `.rpm` or `.deb` file and follow the on-screen
```

Comment:
hm, packages aren't executeable. Its not clear to me whats meant here? were you to point to the instructions on the download page?